src/mtype.d

```diff
@@ -8564,39 +8551,24 @@ public:
         Dsymbol s;
         static if (LOGDOTEXP)
         {
-            printf("TypeClass::dotExp(e='%s', ident='%s')\n", e.toChars(), ident.toChars());
-        }
-        if (e.op == TOKdot)
-        {
-            DotExp de = cast(DotExp)e;
-            if (de.e1.op == TOKscope)
-            {
-                ScopeExp se = cast(ScopeExp)de.e1;
-                s = se.sds.search(e.loc, ident);
-                e = de.e1;
-                goto L1;
-            }
+            printf("TypeClass::dotExp(e = '%s', ident = '%s')\n", e.toChars(), ident.toChars());
```

Comment:
Adding spaces is pointless, and causes problems with line wrap when the expressions are long.